source/manifest.json

```diff
@@ -1,4 +1,10 @@
 {
+  "applications": {
+    "gecko": {
+      "id": "theoldreader@knyar",
+      "strict_min_version": "53.0"
+    }
+  },
```

Comment:
I think this requires more research and can be done outside this PR. For now, just a separate manifest file + `build` folder + `build/*` in `.gitignore` is sufficient.